css/styles.css

```diff
@@ -199,33 +204,294 @@ body {
 }
 
 .social-links {
+  display: flex;
+  flex-direction: column;
   background: #1a2236;
-  height: 15%;
+  padding: 5%;
 }
 
 .resume-link {
   text-decoration: none;
+  display: inline-flex;
+  width: 40%;
 }
 
 .resume-link > div {
   color: #98dede;
-  padding: 5vw;
-  width: 60%;
-  background: url("../res-icons/ic_link_teal.svg") 80% 50% no-repeat;
   font-family: Poppins, sans-serif;
   font-style: normal;
-  font-weight: 500;
+  font-weight: 400;
   font-size: 1.3em;
-  line-height: 1.7em;
+}
+
+.icon-placeholder {
+  width: 20%;
+  background: url("../res-icons/ic_link_teal.svg") 30% 10% no-repeat;
 }
 
 .link-container {
-  height: 10vh;
+  margin-top: 5%;
+  height: 100%;
   width: 100%;
   display: flex;
-  justify-content: space-around;
-}
-
-.link-container > a {
-  height: 60%;
+  justify-content: space-between;
+}
+
+@media screen and (min-width: 768px) {
+  /* styling for navigation bar elements -ram */
+  .nav-header {
+    height: 10vh;
+    display: block;
+    background: #fff;
```

Comment:
- [ ] Great job implementing the feedback from the previous code reviewer to use min-width instead of min-device-width :100:

Reply:
@Debas-31 hi. as i explained to my previous reviewer, it requires using javascript to manidpulate dom, which is not required at this phase of the project. the figma design has two cursors there, which indicates after click, the project details are to be displayed, which again can only be done with javascript. thank you very much for wasting my review and time


![Screenshot_20221208_175513](https://user-images.githubusercontent.com/116718155/206446280-b20156aa-47bc-404d-a06e-c80af58f2882.png)

![image](https://user-images.githubusercontent.com/116718155/206446654-5f892b66-f9b8-4d50-be92-b4a2b2516678.png)